addons/project/security/project_security.xml

```diff
@@ -21,6 +21,7 @@
 
     <record id="group_project_rating" model="res.groups">
         <field name="name">Use Rating on Project</field>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
```

Comment:
idem. Some view fields have groups, so for all combinations to be valid, this groups must involve group_user. Otherwise you must change your access rights and views.